### DISCUSSION TITLE
Adding missing option

### DISCUSSION THREAD

**Discussion by cvrle77** [muraliwebworld/push-notification-plugin-for-wordpress-websites#8] [Ideas]
There is no option to turn off subscription on front page completely. We can only choose between custom, or default, but we can't turn it off, while still being able to send notifications.

**Comment by muraliwebworld** [comment]
Use unsubscribe button when you click on bell prompt it will show unsubscribe button like below, if it is already unsubscribed, it will show subscribe button. **Are you getting below popup?**, it will work only when service worker is registered successfully in your browser 
![Screenshot 2024-05-20 at 6 11 25 PM](https://github.com/muraliwebworld/push-notification-plugin-for-wordpress-websites/assets/32461311/489c6491-cc1f-4302-a0aa-b2e81a05b544)

**Comment by cvrle77** [comment]
You didn't understand me. I was thinking of turning off the subscription prompt from within the plugin so that it doesn't show the subscription prompt to anyone,  but I still want to be able to send notifications to subscribers.
So it's like a checkbox for option 'Enable subscription on front page'

![image](https://github.com/muraliwebworld/push-notification-plugin-for-wordpress-websites/assets/16167396/00340b9c-806f-4df5-a577-e7d35ef90919)

**Comment by muraliwebworld** [comment]
I will look into this...once i complete pwa related things, i will look into this
